build.html

```diff
@@ -3,8 +3,8 @@
   <div class="verfication-offline">Your device is offline</div>
   <div class="state" v-bind:class="{ start: loading, past: verifyCode || confirmation, present: auth, 'has-code': storedEmail}" data-state="auth">
     <form v-on:submit.prevent="sendValidation">
-      <h2 class="aligncenter">Authentication</h2>
-      <p>Authenticate your email address below to gain access to the app.</p>
+      <h2 class="aligncenter">Verification</h2>
+      <p>Verify your email address below to gain access.</p>
```

Comment:
I think you used the term `to process` in the email verification one?

Best to make sure all the phrasings are in sync as much as possible. Here it is for your reference https://github.com/Fliplet/fliplet-widget-email-verification/pull/13/files

Reply:
I think it's actually the same? I don't see that I've used that term anywhere.